### ISSUE TITLE
[Bug] Pages only load on second try when browser is closed

### ISSUE THREAD

**Issue by mobd** [mozilla-mobile/fenix#19562]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
Completely close the browser and click on a link in a different app

### Expected behavior
Page is loaded

### Actual behavior
I get a blank screen. Only after reloading does the page appear

### Device information

* Device vendor / model and Android version: Huawei Nova 6 (Android 10)
* Firefox for Android version: 90.0a1 (Build #2015811147)

**Comment by SoftVision-LorandJanos** [comment]
Hi @mobd ! Thank you for your report! I cannot reproduce the issue as described above. I have set the browser as the default browser, closed it completely and opened a link from another app. The page does turn completely white for a few seconds, but then the page loads as expected without reloading manually. Since the cause of the issue appears to be the same as on [#19563](https://github.com/mozilla-mobile/fenix/issues/19563), I will close this one as duplicate.
Tested using:
Device vendor / model and Android version: Motorola Nexus 6 (Android 7.1.1).
Firefox for Android version: 90.0a1 (Build #2015811147) and 90.0a1 (Build #2015811433).
